runtime/browser/xwalk_browser_context.h

```diff
@@ -146,6 +146,7 @@ class XWalkBrowserContext
   PartitionPathContextGetterMap context_getters_;
   std::unique_ptr<XWalkSSLHostStateDelegate> ssl_host_state_delegate_;
   std::unique_ptr<content::PermissionManager> permission_manager_;
+  scoped_refptr<XWalkSpecialStoragePolicy> special_storage_policy_;
```

Comment:
You forgot to include the appropriate header declaring `XWalkSpecialStoragePolicy`...